src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,23 @@
+
+.TodoInfo {
+  width: 200px;
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: antiquewhite;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```

Comment:
Avoid using multiple font families in a single element, it can lead to unintended behavior and inconsistency. 🤔 Consider using a single font family and adding a fallback. Example: font-family: Arial, sans-serif;